lib/appsignal/utils/params_sanitizer.rb

```diff
@@ -6,42 +6,41 @@ class ParamsSanitizer
 
       class << self
         def sanitize(params, options = {})
-          sanitize_value(params, options)
+          sanitize_value(params, options.fetch(:filter_parameters, []))
+        end
+
+        def sanitize_hash(source, filter_parameter_keys)
```

Comment:
Since this is a private Util class I don't think we need to keep backwards compatibility and put it in a separate method. If anyone disagrees, we should at least print a deprecation warning.